common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/TimeLogger.cs

```diff
@@ -0,0 +1,53 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Services.Diagnostics {
```

Comment:
Remove "services".   If this is a utility class, add to Utils namespace instead.   See extension comment, the extension could use the utility class internally.